docs/.eleventy.js

```diff
@@ -193,7 +182,7 @@ module.exports = function(eleventyConfig) {
     const markdownIt = require("markdown-it");
     const md = markdownIt({ html: true, linkify: true, typographer: true, highlight: (str, lang) => highlighter(md, str, lang) })
         .use(markdownItAnchor, {
-            slugify
+            slugify: slug
```

Comment:
```suggestion
            slugify: s => slug(s)
```

To ensure that the `slug` function is always passed one argument, because it [accepts two](https://github.com/Flet/github-slugger/blob/4bfc760a4702f0a4ea4c1d35dc6d1d865ca119a5/index.js#L44-L50).